src/components/Join/Join.tsx

```diff
@@ -0,0 +1,179 @@
+import { Button } from 'components'
+import { ExternalButtonLink } from 'components/Button/Button'
+import { oidcIssuers } from 'config'
+import { ChangeEvent, Fragment, useState } from 'react'
+import Modal from 'react-modal'
+import styles from './Join.module.scss'
+
+const tabs = [
+  {
+    id: 'show-options',
+    label: 'Show me some providers!',
+    content: (
+      <>
+        Here are some Pod providers that work with sleepy.bike:
+        <ul>
+          {oidcIssuers.map(({ issuer, registration }) => (
+            <li>
+              <ExternalButtonLink
+                secondary
+                href={registration}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {new URL(issuer).hostname}
+              </ExternalButtonLink>
+            </li>
+          ))}
+        </ul>
+      </>
+    ),
+  },
+  {
+    id: 'choose-for-me',
+    label: 'Choose for me!',
+    content: (
+      <>
+        <ExternalButtonLink
+          secondary
+          href="https://solidweb.me/idp/register/"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Get a Pod at solidweb.me 😉
+        </ExternalButtonLink>
+        <br />
+        It runs the more modern Community Solid Server
```

Comment:
I am not sure if _the more modern_ is grammatically correct? I'd say rather _the most modern_ or _more modern CSS than \<something else>_.

Reply:
Tried to clarify. Hope it's better now.